blockchain/blockchain.go

```diff
@@ -272,6 +283,12 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 			}
 		}
 	}
+
+	for i := 1; i <= runtime.NumCPU()/2; i++ {
+		bc.wg.Add(1)
+		go bc.prefetchTxWorker(i)
+	}
+
 	// Take ownership of this particular state
 	go bc.update()
 	go bc.gcCachedNodeLoop()
```

Comment:
```suggestion
	bc.gcCachedNodeLoop()
```
@ehnuje This line raise a race condition between `bc.wg.Done()` and `bc.wg.Add(1)` in `bc.gcCachedNodeLoop()`